modules/genomics_cromwell/README.md

```diff
@@ -0,0 +1,35 @@
+# RAD Lab Genomics-Cromwell Module
+
+## GCP Products/Services 
+
+* Life Sciences API
+* Cloud Compute
+* CloudSQL
+* Cloud Storage
+* Virtual Private Cloud (VPC)
+
+## Module Overview 
+Cromwell is a Workflow Management System geared towards scientific workflows. Cromwell is open sourced under the BSD 3-Clause license. Checkout Cromwell documentation at https://cromwell.readthedocs.io/
```

Comment:
Can you add a clickable URL?

Reply:
Done, will be included in the next commit